configs/examples/grpo_letter_counting/gcp_job.yaml

```
@@ -0,0 +1,48 @@
+# Job config to GRPO tune a letter counting model.
+#
+# Requirements:
+#   - Set up SkyPilot GCP: https://oumi.ai/docs/en/latest/user_guides/launch/launch.html#setup
+#   - Log into WandB (`wandb login`) or disable `enable_wandb`
+#
+# Usage:
+#   oumi launch up -c configs/examples/grpo_letter_counting/gcp_job.yaml --cluster letter-counting-grpo
+#
+# See Also:
+#   - Documentation: https://oumi.ai/docs/en/latest/user_guides/launch/launch.html
+#   - Config class: oumi.core.configs.JobConfig
+#   - Config source: https://github.com/oumi-ai/oumi/blob/main/src/oumi/core/configs/job_config.py
+#   - Other job configs: configs/**/*job.yaml
+
+name: letter-counting-grpo
+
+resources:
+  cloud: gcp
+  accelerators: "A100:4"
+  use_spot: false
+
+working_dir: .
+
+file_mounts:
+  ~/.netrc: ~/.netrc  # WandB credentials
+  ~/.cache/huggingface/token: ~/.cache/huggingface/token # HF credentials
+
+envs:
+  WANDB_PROJECT: oumi-train
+  OUMI_RUN_NAME: letter-counting.grpo
+
+setup: |
+  set -e
+  pip install uv && uv pip install oumi[gpu] vllm
+  pip install -U flash-attn --no-build-isolation
+
+run: |
+  set -e  # Exit if any command failed.
+  source ./configs/examples/misc/sky_init.sh
+
+  set -x
+  oumi distributed torchrun \
+      -m oumi train \
+      -c configs/examples/grpo_letter_counting/train.yaml \
```

We may consider starting to prefix these scripts with `oumi://` so they can use this example without having the github repo downloaded locally.
```suggestion
      -c oumi://configs/examples/grpo_letter_counting/train.yaml \
```

---

Is there an issue filed for this? IMO that change should be done in a separate PR which modifies all config files at once (your favorite type of PR :p ). Much easier to track where we've changed it or not

---

No need to update all configs right now! I'd rather you use it here if we plan on making a blog post for this config though.

---

I can quickly submit a PR to update them all before the blog post (we should get better results for this before posting). I'd rather have configs changes be all-or-none, so people don't find our configs in an inconsistent state.